plugins/block-extension-tooltip/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@blockly/block-extension-tooltip",
+  "version": "0.0.1",
+  "description": "A Blockly block extension that adds support for custom tooltip rendering.",
+  "scripts": {
+    "audit:fix": "blockly-scripts auditFix",
+    "build": "blockly-scripts build",
+    "clean": "blockly-scripts clean",
+    "dist": "blockly-scripts build prod",
+    "lint": "blockly-scripts lint",
+    "prepublishOnly": "npm run clean && npm run dist",
```

Comment:
Also needs a predeploy script. Is that not being added in create-package?

Reply:
Done, and added to the create-package script.